### DISCUSSION TITLE
Typing animation

### DISCUSSION THREAD

**Discussion by NS-Salt** [remarkjs/remark#1321] [Q&A] [answered]
Hi, i'm currently using this plugin in my project and I'm looking for a way to add typing animation, any suggestions?

**Comment by ChristianMurphy** [comment, marked as answer]
Welcome @NS-Salt! 👋

I'm not sure what "this plugin" refers to there are hundreds in the remark ecosystem.

> I'm looking for a way to add typing animation, any suggestions?

Use CSS. For example https://css-tricks.com/snippets/css/typewriter-effect/
